xdsl/irdl/constraints.py

```diff
@@ -20,6 +20,9 @@ class ConstraintContext:
     variables: dict[str, Attribute] = field(default_factory=dict)
     """The assignment of constraint variables."""
 
+    ranges: dict[str, tuple[Attribute, ...]] = field(default_factory=dict)
+    """The constraint range variables."""
```

Comment:
is this also an assignment?

Reply:
Would be good to have the docstrings be consistent if yes

Reply:
Have changed the docstring